packages/paste-core/components/checkbox/src/Checkbox.tsx

```diff
@@ -64,7 +64,16 @@ const CheckboxIcon: React.FC<{
   if (indeterminate) {
     return <MinusIcon element={element} decorative color={color} size="sizeIcon10" />;
   }
-  return <CheckboxCheckIcon element={element} hidden={!checked} decorative color={color} size="sizeIcon10" />;
+  return (
+    <CheckboxCheckIcon
+      element={element}
+      hidden={!checked}
```

Comment:
So, the bug is on this line. `hidden` isn't being set on the svg element in the DOM. The logic is already there. Needs a test

Reply:
Seems like no icons use the `hidden` prop... they all use `display`. I can just take out the `hidden` line?

Reply:
Sure, lets still have a test though, hey?

Reply:
yeee adding them rn